repos/system_upgrade/el7toel8/actors/bootentriesscanner/actor.py

```diff
@@ -0,0 +1,18 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.bootentriesscanner import scan_boot_entries
+from leapp.models import BootEntry
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+
+class Bootentriesscanner(Actor):
+    """
+    Scans the bootloader configuration for available boot entries.
+    """
+
+    name = 'boot_entries_scanner'
+    consumes = ()
+    produces = (BootEntry, )
```

Comment:
```suggestion
    produces = (BootEntry,)
```

Reply:
This was implemented in the new version. Thanks!